wrappers/python/tests/layer/test_llama_attention.py

```diff
@@ -211,3 +214,21 @@ def test_forward_backward(self, starpu_simple, torch_rng, dtype: str,
             if p1.requires_grad:
                 g1, g2 = p1.grad, p2.grad
                 assert torch.norm(g1 - g2) <= rtol * torch.norm(g1)
+
+    def test_flops_counting(self, starpu_simple, torch_rng, dtype: str,
+                            params: LlamaAttentionTestParams, bias: bool):
+
+        _, nntile_layer, *_ = \
+                generate_inputs(dtype, params, bias)
+        nntile_layer.forward_async()
+        nntile_layer.backward_async()
```

Comment:
we do not need these forward and backward operations, as they are not tested here